protean/detail/data_table_column.ipp

```diff
@@ -96,7 +96,7 @@ namespace protean { namespace detail {
         { push_back_impl_tmpl<type>(value, p); }
 
     template <variant_base::enum_type_t E>
-    void data_table_column<E>::push_back_impl(const variant& value, variant* p = 0)
+    void data_table_column<E>::push_back_impl(const variant& value, variant* p/* = 0*/)
```

Comment:
Why not just remove rather than commenting?

Reply:
Commenting defaults is the style used elsewhere in the project.

Reply:
OK. Not nice but probably better to be consistent.

Reply:
It's quite common to do this as 'documentation' of what the default value is. (I've never seen the purpose really.)

Ought to be using `nullptr` of course :)